amr-wind/equation_systems/icns/icns_advection.H

```diff
@@ -190,11 +190,14 @@ struct AdvectionOp<ICNS, fvm::Godunov>
                 m_allow_inflow_on_outflow = true;
             }
 
+            // if state is NPH, then n and n+1 are known, and only
+            // spatial extrapolation is performed
+            amrex::Real dt_arg = (fstate == FieldState::NPH) ? 0.0 : dt;
```

Comment:
let's give this a better name than `dt_arg`. `dt_for_hydro` ? `dt_local` ? `dt_adjusted`?

Reply:
Maybe dt_extrap as in dt for extrapolation?

Reply:
Changed this into dt_extrap, as in the dt that is used for extrapolation.